server/src/test/java/com/paypal/selion/grid/servlets/SauceServletTest.java

```diff
@@ -69,10 +74,15 @@ public void testDoGet() throws Exception {
         MockHttpServletResponse response = new MockHttpServletResponse();
         servlet.doGet(request, response);
         validateHtmlResponseContent(response, "Grid Management Console", "Sauce node registered successfully");
+        assertSame(hub.getRegistry(), servlet.getRegistry(), "hub and servlet do not point to the same registry");
+        System.err.println("---- Active proxies (hub):" + hub.getRegistry().getAllProxies().getSorted());
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [System.err.println is used](https://www.codacy.com/app/SeLion/paypal-SeLion/file/2360831189/issues/source?bid=2928425&fileBranchId=3219779#l78)